src/core.js

```diff
@@ -24,6 +23,10 @@ const test = (name, callback) => {
   }
 };
 
+Object.defineProperty(test, 'skip', {value: ()=> {}, writable: true});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/codingstyle-cn/hlj/pullRequest?prid=7143429)
```suggestion
Object.defineProperty(test, "skip", {value: ()=> {}, writable: true});
```